packages/engine/Source/Core/HeightmapTerrainData.js

```diff
@@ -94,16 +94,12 @@ import TerrainProvider from "./TerrainProvider.js";
  * @see GoogleEarthEnterpriseTerrainData
  */
 function HeightmapTerrainData(options) {
+  options = options ?? Frozen.EMPTY_OBJECT;
+
   //>>includeStart('debug', pragmas.debug);
-  if (!defined(options) || !defined(options.buffer)) {
-    throw new DeveloperError("options.buffer is required.");
-  }
-  if (!defined(options.width)) {
-    throw new DeveloperError("options.width is required.");
-  }
-  if (!defined(options.height)) {
-    throw new DeveloperError("options.height is required.");
-  }
+  Check.defined("options.buffer", options.buffer);
```

Comment:
I think this is equivalent to:

```js
//>>includeStart('debug', pragmas.debug);
Check.defined("options", options);
Check.defined("options.buffer", options.buffer);
```
Or maybe even better `Check.typeOf.object("options", options)`